### PR TITLE
Reworking UU matter

### PR DESCRIPTION
**Pull request by Ayutac**
Tell me what you think of it so far :)

1.19 version of #2959

### CODE REVIEW

**Comment by Ayutac** [comment]
I'm satisfied so far. Some uu recipes got changed, but this was inevitable. The idea is that adding non-uu to the recipes makes for much more available recipes in the less-uu-requiring category. Lore-wise uu matter works well if it can connect to some existing matter.

Since I made a sea lanterns recipe I got rid of the "grind shards -> crystals" recipe I loathed for some time. The reason is that I want to keep glowstone and sea crystals as two different sources of light and since shards don't emit light, the recipe made no sense on the lore I built up.

Oh, and the reason to add one uu-matter to redstone ore is because redstone is so damn important to TR.

**Review by drcrazy — CHANGES_REQUESTED** [review]
Please, fix names and namespaces

File: src/main/resources/data/techreborn/recipes/crafting_table/uu_matter/wood/arcadia_wood.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "key": {
+	"S": {
+      "tag": "c:saw_dusts"
+	},
+    "U": {
+      "item": "techreborn:uu_matter"
+    }
+  },
+  "result": {
+    "item": "minecraft:arcadia_wood"
```
Comment (drcrazy) [inline src/main/resources/data/techreborn/recipes/crafting_table/uu_matter/wood/arcadia_wood.json]: Is it acacia?

File: src/main/resources/data/techreborn/recipes/crafting_table/uu_matter/ore/coal_ore.json [hunk 2]
```diff
@@ -3,9 +3,12 @@
   "pattern": [
     "U  ",
     " U ",
-    "   "
+    " S "
   ],
   "key": {
+    "S": {
+      "item": "techreborn:stone"
```
Comment (drcrazy) [inline src/main/resources/data/techreborn/recipes/crafting_table/uu_matter/ore/coal_ore.json]: minecraft namespace

File: src/main/resources/data/techreborn/recipes/crafting_table/uu_matter/sapling/birch_sapling.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "type": "reborncore:padded",
+  "pattern": [
+    "  U",
+    " P ",
+    " S "
+  ],
+  "key": {
+    "P": {
+      "item": "techreborn:plant_ball"
```
Comment (drcrazy) [inline src/main/resources/data/techreborn/recipes/crafting_table/uu_matter/sapling/birch_sapling.json]: plantball, without underscore

**Comment by Ayutac** [comment]
Really weird, did not show these errors earlier. Will correct once i get home today :)

**Pull request merged**